docs/mkdocs.yml

```diff
@@ -65,9 +69,7 @@ nav:
       - cmdref/spiderpool-controller.md
       - cmdref/spiderpool-agent.md
   - Development:
-      - develop/pullrequest.md
+      - develop/roadmap.md
+      - develop/contributing.md
       - develop/release.md
-      - develop/swagger_openapi.md
-      - develop/test.md
-      - develop/changelog.md
       - develop/upgrade.md
```

Comment:
The code patch adds and removes menu items from a website navigation, so there's no particular bug risk involved. Suggestions for improvement could be:

- It seems that some files were renamed, so you may want to check the links and references to these files in other parts of the codebase to make sure everything still works.
- You could consider adding more descriptive names to the new menu items, e.g., instead of "get-started-sriov.md," you could use something like "using SR-IOV networking."
- The "debug.md" item appears twice in the list, you may want to remove one of them or rename them to be more specific.
- It might be useful to add a link to the project's source code repository or issue tracker in the "Development" section, which would make it easier for developers to contribute and track changes.